tests/test_fips.py

```diff
@@ -195,6 +195,10 @@ def test_gnutls_binary(container_per_test: ContainerData) -> None:
         "mv fips-test-gnutls /bin/fips-test-gnutls"
     )
 
+    assert "NOT" not in container_per_test.connection.check_output(
```

Comment:
This check is wrong. The requirement states:

`should return "library is in FIPS140-3 mode"or "library is in FIPS140-2 mode" or "library is in FIPS140 mode"`

So just checking for NOT is not what the requirement states.

Reply:
Changed :)